### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by SteveGremory** [hecrj/iced#835] [Ideas]
Hey People! I find iced goddamn amazing and even though this isn't the most pressing issue, i'd very much love to have an iced discord server for better, more efficient communication.
Thanks!

**Comment by 13r0ck** [comment]
Have you tried the [Zulip](https://iced.zulipchat.com/login/) chat? Personally I like it more than Discord for this kind of thing.

Though I am not sure if you have ever been over to the [Elm](https://elm-lang.org/) slack, It is kinda a disaster for many reasons because they are a larger community. And I am concerned that as Iced grows it will run into the same issues whether on Zulip/Discord/Slack/etc

**Comment by 13r0ck** [comment]
Actually an interesting solution posted over there was to create a bot that would periodically remind people to post their Q and As over on Stack Overflow to make it possible for future people to find solutions to their questions via search engine.

I actually really like this. If Iced were to implement this then I am fine with both Zulip/Discord/etc or even both. But I am sure Hecrj is familiar with Discord and has chosen Zulip for a reason.

**Comment by SteveGremory** [comment]
The Zulip UI is..Well.
It's- Mediocre. The Stack idea is great tho, and can be implemented with a discord bot so that every time someone wants to ask a question, they can just `!question` or `[Q]` then their question and the bot will create a stack post alongside a UUID, the person who wants to answer can just simply just reply to them or be like `!answer <question_message_id> <answer>`

**Comment by 13r0ck** [comment]
Ya UI isn't great, but It has some great features that Discord doesn't have, like referencing a Github number like `#827` automatically converts that into a link, and the conversation categories are great for finding information from past conversations, neither of those Discord has. Zulip does also have bots, just Iced's Zuliip chat is so small that we don't need them yet.

Just give Zulip a try, and If you still hate it then we can talk more :smile:

**Comment by Joe23232** [comment]
Is there a discord/zulip server for Iced?

**Comment by 13r0ck** [comment]
@Joe23232 Sure is! 
https://iced.zulipchat.com/

**Comment by Joe23232** [comment]
Thank you mate :)

**Comment by hecrj** [comment]
I'm open to trying a Discord server if there is enough interest.
